cansat-stm32f446/src/blink.rs

```diff
@@ -0,0 +1,28 @@
+//! Blinking LED using  blocking `Delay` in  TIM5.
+#![deny(unsafe_code)]
```

Comment:
This is already defined in the main.rs. Since `mod blink` is nested inside main, the same rule applies also here.
```suggestion
```